lib/src/base-styles/_base.scss

```diff
@@ -20,6 +20,7 @@
 @import '../scss-shared/nx-container-helpers';
 @import '../scss-shared/nx-text-helpers';
 @import '../scss-shared/nx-flexbox-helpers';
+@import '../scss-shared/nx-mask-helpers';
```

Comment:
I don't think there's actually a need to import this here.  There only would be if it was being used somewhere within the base styles.

Reply:
> I don't think there's actually a need to import this here. There only would be if it was being used somewhere within the base styles.

You're right: https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/3f9802b29f8688333f596400318d9b09feb2d7bb